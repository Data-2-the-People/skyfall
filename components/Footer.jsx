import React from 'react'

const Footer = () => (
  <footer>
    <a href="https://vercel.com?utm_source=data2thepeople-org&utm_campaign=oss">
      <img
        src="/powered-by-vercel.svg"
        alt="Powered by Vercel"
        className="logo"
      />
    </a>
    <style jsx>{`
      footer {
        width: 100%;
        height: 100px;
        border-top: 1px solid #eaeaea;
        display: flex;
        justify-content: center;
        align-items: center;
      }
      footer img.logo {
        margin-left: 0.5rem;
        height: 2em;
      }
      footer a {
        display: flex;
        justify-content: center;
        align-items: center;
      }
    `}</style>
  </footer>
)

export default Footer
