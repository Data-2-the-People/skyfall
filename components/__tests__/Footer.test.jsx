import React from 'react'
import { render } from 'enzyme'
import Footer from '../Footer'

describe('Footer', () => {
  it('renders Powered by Vercel', () => {
    expect.assertions(1)
    const rendered = render(<Footer />)
    // https://vercel.com/knowledge/can-vercel-sponsor-my-project
    expect(rendered).toMatchSnapshot()
  })
})
