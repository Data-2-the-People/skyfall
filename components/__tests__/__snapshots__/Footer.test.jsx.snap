// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer renders Powered by Vercel 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "jsx-891755636",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "class": "jsx-891755636",
          "href": "https://vercel.com?utm_source=data2thepeople-org&utm_campaign=oss",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "alt": "Powered by Vercel",
              "class": "jsx-891755636 logo",
              "src": "/powered-by-vercel.svg",
            },
            "children": Array [],
            "name": "img",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "alt": undefined,
              "class": undefined,
              "src": undefined,
            },
            "x-attribsPrefix": Object {
              "alt": undefined,
              "class": undefined,
              "src": undefined,
            },
          },
        ],
        "name": "a",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
          "href": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
          "href": undefined,
        },
      },
    ],
    "name": "footer",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
